### PR TITLE
feat: codesniffer and rector fixes for Drupal 10

### PR DESCRIPTION
**Pull request by podarok**
Addresses: https://github.com/YCloudYUSA/yusaopeny/pull/8
Fixes: https://yusa.atlassian.net/browse/DS-64

### CODE REVIEW

File: openy_prgf/modules/openy_prgf_event_listing/openy_prgf_event_listing.module [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
+
 /**
  * Implements hook_form_FORM_ID_alter().
  */
-function openy_prgf_event_listing_form_views_exposed_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state) {
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ *
```
Comment (andreyzb) [inline openy_prgf/modules/openy_prgf_event_listing/openy_prgf_event_listing.module:14]: Empty docblock

File: openy_prgf/modules/openy_prgf_class_sessions/src/Plugin/Block/ClassSessions.php [hunk 2]
```diff
@@ -119,6 +119,9 @@ public function build() {
     return $class_sessions;
   }
 
+  /**
+   *
```
Comment (andreyzb) [inline openy_prgf/modules/openy_prgf_class_sessions/src/Plugin/Block/ClassSessions.php:123]: Empty docblock

File: openy_egym_iwatch/openy_egym_iwatch.install [hunk 3]
```diff
@@ -11,22 +15,25 @@ use Drupal\Core\File\FileSystemInterface;
  * Open Y eGym iWatch install file.
  */
 
+/**
+ *
```
Comment (andreyzb) [inline openy_egym_iwatch/openy_egym_iwatch.install:19]: Empty docblock

File: openy_node/modules/openy_node_blog/src/BlogSimplePaginator.php [hunk 4]
```diff
@@ -6,6 +6,9 @@
 use Drupal\Core\Link;
 use Drupal\Core\Routing\RouteMatchInterface;
 
+/**
+ *
```
Comment (andreyzb) [inline openy_node/modules/openy_node_blog/src/BlogSimplePaginator.php:10]: Empty docblock

File: openy_node/modules/openy_node_social_post/openy_node_social_post.install [hunk 5]
```diff
@@ -5,6 +5,9 @@
  * Open Y Node Social Post install file.
  */
 
+/**
+ *
```
Comment (andreyzb) [inline openy_node/modules/openy_node_social_post/openy_node_social_post.install:9]: Empty docblock

File: openy_node/src/BatchNodeUpdate.php [hunk 6]
```diff
@@ -1,12 +1,10 @@
...
 
-/**
- * @file
- * OpenY Node batch update class.
- */
-
 namespace Drupal\openy_node;
 
+/**
+ *
```
Comment (andreyzb) [inline openy_node/src/BatchNodeUpdate.php:6]: Empty docblock

File: openy_prgf/modules/openy_prgf_embedded_groupexpro_schedule/openy_prgf_embedded_groupexpro_schedule.install [hunk 7]
```diff
@@ -35,7 +35,8 @@ function openy_prgf_embedded_groupexpro_schedule_update_8001() {
  * Enable openy_gxp if not enabled.
  */
 function openy_prgf_embedded_groupexpro_schedule_update_8002() {
-  $module_handler = \Drupal::moduleHandler();;
+  $module_handler = \Drupal::moduleHandler();
+  ;
```
Comment (andreyzb) [inline openy_prgf/modules/openy_prgf_embedded_groupexpro_schedule/openy_prgf_embedded_groupexpro_schedule.install:39]: duplicated semicolon?

File: openy_prgf/modules/openy_prgf_group_schedules/openy_prgf_group_schedules.install [hunk 8]
```diff
@@ -34,7 +34,8 @@ function openy_prgf_group_schedules_update_8001() {
  * Enable openy_gxp if not enabled.
  */
 function openy_prgf_group_schedules_update_8002() {
-  $module_handler = \Drupal::moduleHandler();;
+  $module_handler = \Drupal::moduleHandler();
+  ;
```
Comment (andreyzb) [inline openy_prgf/modules/openy_prgf_group_schedules/openy_prgf_group_schedules.install:38]: Duplicate semicolon?

File: openy_node/modules/openy_node_blog/src/BlogSimplePaginator.php [hunk 9]
```diff
@@ -40,28 +43,28 @@ public function __construct(EntityTypeManagerInterface $entity_type_manager, Rou
...
-                 ->condition('status', 1)
-                 ->condition('langcode', $currentNode->language()->getId())
-                 ->sort('created', $sort)
-                 ->range(0, 1)
-                 ->execute();
+      ->condition('type', $currentNode->bundle())
+      ->condition('status', 1)
+      ->condition('langcode', $currentNode->language()->getId())
+      ->sort('created', $sort)
+      ->range(0, 1)
```
Comment (froboy) [inline openy_node/modules/openy_node_blog/src/BlogSimplePaginator.php:66]: This query needs an access check as per [this change record](https://www.drupal.org/node/3201242).
```suggestion
      ->range(0, 1)
      ->accessCheck(FALSE)
```

**Review by froboy — CHANGES_REQUESTED** [review]
@podarok I found one function missing an accessCheck that threw an error in D10. I can submit that separately, but it seems relevant in the context of these fixes. What else do we need to get them completed?

File: openy_prgf/modules/openy_prgf_group_schedules/openy_prgf_group_schedules.install (hunk 8 above)
Comment (froboy) [reply openy_prgf/modules/openy_prgf_group_schedules/openy_prgf_group_schedules.install:38]:
```suggestion
```

File: openy_prgf/modules/openy_prgf_embedded_groupexpro_schedule/openy_prgf_embedded_groupexpro_schedule.install (hunk 7 above)
Comment (froboy) [reply openy_prgf/modules/openy_prgf_embedded_groupexpro_schedule/openy_prgf_embedded_groupexpro_schedule.install:39]:
```suggestion
```

File: openy_node/src/BatchNodeUpdate.php [hunk 10]
```diff
@@ -1,12 +1,10 @@
...
+ */
```
Comment (froboy) [inline openy_node/src/BatchNodeUpdate.php:7]:
```suggestion
```

File: openy_node/modules/openy_node_social_post/openy_node_social_post.install [hunk 11]
```diff
@@ -5,6 +5,9 @@
...
+ */
```
Comment (froboy) [inline openy_node/modules/openy_node_social_post/openy_node_social_post.install:10]:
```suggestion
```

File: openy_egym_iwatch/openy_egym_iwatch.install [hunk 12]
```diff
@@ -11,22 +15,25 @@ use Drupal\Core\File\FileSystemInterface;
...
+ */
```
Comment (froboy) [inline openy_egym_iwatch/openy_egym_iwatch.install:20]:
```suggestion
```

File: openy_node/modules/openy_node_blog/src/BlogSimplePaginator.php [hunk 13]
```diff
@@ -6,6 +6,9 @@
 use Drupal\Core\Link;
 use Drupal\Core\Routing\RouteMatchInterface;
 
+/**
+ *
+ */
```
Comment (froboy) [inline openy_node/modules/openy_node_blog/src/BlogSimplePaginator.php:11]:
```suggestion
```

File: openy_prgf/modules/openy_prgf_class_sessions/src/Plugin/Block/ClassSessions.php [hunk 14]
```diff
@@ -119,6 +119,9 @@ public function build() {
...
+   */
```
Comment (froboy) [inline openy_prgf/modules/openy_prgf_class_sessions/src/Plugin/Block/ClassSessions.php:124]:
```suggestion
```

File: openy_prgf/modules/openy_prgf_event_listing/openy_prgf_event_listing.module [hunk 15]
```diff
@@ -1,18 +1,28 @@
...
+ */
```
Comment (froboy) [inline openy_prgf/modules/openy_prgf_event_listing/openy_prgf_event_listing.module:15]:
```suggestion
```

**Review by froboy — CHANGES_REQUESTED** [review]
Added suggestions to @andreyzb's comments.

**Pull request merged**